composer.json

```
@@ -24,7 +24,7 @@
         "silverstripe/vendor-plugin": "^2",
         "symfony/filesystem": "^6.1",
         "intervention/image": "^2.7.2",
-        "league/flysystem": "^1.1.9"
+        "league/flysystem": "^3.0"
```

```suggestion
        "league/flysystem": "^3.9.0"
```
Latest release is 3.9.0 https://github.com/thephpleague/flysystem/releases/tag/3.9.0

---

If we use `^` before version number, composer should install a latest release `>=3.0 < 4.0`.
I checked the version after install and it's 3.9.0

---

Use `^3.9.0`

The reason is that we run a `composer update --prefer-lowest` job in our CI - [example](https://github.com/silverstripe/silverstripe-framework/actions/runs/3321789438/jobs/5489921340).  Under this job, `3.0.0` would be installed.  I see no reason to support older versions so we may as well set the minimum to `3.9.0`

---

DONE